userspace/libsinsp/sinsp_ppm_sc.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <sinsp.h>
+
+std::unordered_set<uint32_t> sinsp::simple_ppm_sc_set()
```

Comment:
Why `simple`? 
Perhaps `default`?

@falcosecurity/libs-maintainers @falcosecurity/falco-maintainers wdyt?

Reply:
We should also add a comment to explain why one should use this set.

Reply:
Because we already got a `default` that is "catch everything"; instead, given this is a subset, i'd rather keep the old `simple-consumer` naming, but without the consumer part.

Reply:
ok it makes sense :+1:

Reply:
Added some docs :)

Reply:
yeah simple seems ok :)